### PR TITLE
Recipe(or other) RecyclerView

### PR DESCRIPTION
**Pull request by lytvynov-ihor**
On main page(after login) created a RecyclerView list

You are able to find a list layout and it contains:
images
2 buttons -- share/like etc

### CODE REVIEW

File: app/src/main/java/com/example/mobaplicationproject/ListRecycleActivity.kt
Comment (Lemon049) [inline app/src/main/java/com/example/mobaplicationproject/ListRecycleActivity.kt:1]: good code

File: app/src/main/java/com/example/mobaplicationproject/RegisterFragment.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import com.example.xyz.CredentialsManager
+import com.google.android.material.textfield.TextInputLayout
+
+class RegisterFragment : Fragment(R.layout.register_screen) {
+    private val credentialsManager = CredentialsManager()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val logIn = view.findViewById<TextView>(R.id.LogInTxt)
```
Comment (Lemon049) [inline app/src/main/java/com/example/mobaplicationproject/RegisterFragment.kt:17]: you could change naming for better understanding

File: app/src/main/res/layout/fragment_a.xml
Comment (Lemon049) [inline app/src/main/res/layout/fragment_a.xml:1]: Nice renaming

**Review by Lemon049 — APPROVED** [review]
I would suggest some renaming of variables for better readability, but everything works as intended

**Review by neponimanie — APPROVED** [review]
2 layouts were added (1 for RecyclerView and 1 for an item) as well as two scripts to display the items on the RecyclerView. The list is displayed, each item and button is interactable and shows a toast with item id.

**Pull request merged**